react/Autosuggest/Autosuggest.js

```diff
@@ -68,9 +81,28 @@ export default class Autosuggest extends Component {
     );
   }
 
+  scrollOnFocus = () => {
+    // Temporary solution till we do CSS in JS
+    const getMatchMedia = invoke(window, 'matchMedia', smallDeviceOnlyMedia);
+    const isMobileWidth = get(getMatchMedia, 'matches');
+
+    if (isMobileWidth) {
+      this.textField.scrollIntoView();
+    }
+  }
+
   renderInputComponent(inputProps) {
+    const onFocus = () => {
+      this.scrollOnFocus();
+      inputProps.onFocus(event);
```

Comment:
We should probably be checking that the function exists first, or this could throw.